config/v1/types_node.go

```diff
@@ -77,6 +77,7 @@ type NodeStatus struct {
 }
 
 // +kubebuilder:validation:Enum=v1;v2;""
+// +kubebuilder:validation:XValidation:rule="self != \"v1\"",message="cgroups v1 is not supported on openshift anymore"
```

Comment:
What have we done to existing clusters to ensure that no cluster has the v1 value?

Reply:
The plan is as follows:
- Support v1 till 4.18
- Set the upgradeability of Machine Config operator to "False" if a cluster is found to be on v1 and ask the customer to update the cgroupMode to v2
- Back port this change to 4.18 and ensure every cluster upgrades to this specific version i.e. 4.18.z before upgrading to 4.19